src/components/ResourcesSection/index.jsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import { Col, Row } from "antd";
+import "./index.css";
+import "antd/dist/antd.css";
+import { Link, BrowserRouter as Router } from "react-router-dom";
+import { useTranslation } from "react-i18next";
+import recycleImg from "./../../images/recycle.svg";
+import earthImg from "./../../images/earth.svg";
+import pollutionImg from "./../../images/pollution.svg";
+
+const ResourcesSection = () => {
+  const [t] = useTranslation();
+
+  return (
+    // "site-card-wrapper" is a built in ant Designe className
+    <div className="site-card-wrapper">
```

Comment:
For BEM standardized naming, instead of using dashes in class naming you may use camelCase and/or double underscores. 
e.g `siteCard__wrapper`

Reply:
This is Ant-design class name, that's why it's written like that.